daml-foundations/daml-tools/da-hs-daml-cli/DA/Cli/Visual.hs

```diff
@@ -22,21 +22,18 @@ import qualified Data.ByteString as B
 import Codec.Archive.Zip
 import System.FilePath
 import qualified Data.Map as M
-import Data.Word
-import qualified Data.HashMap.Strict as DHM
-import qualified Data.ByteString.Char8 as CH
+import qualified Data.HashMap.Strict as Map
+import qualified Data.ByteString.Char8 as BS
 import qualified Data.List.Split as DLS
-import qualified Data.List as DL
-import qualified Data.Text as T
+import Data.List
```

Comment:
There's no need to import `Data.List` when you import `Data.List.Extra` since the latter re-exports the former.

Reply:
I am surprised that Hlint did catch that.

Reply:
https://github.com/ndmitchell/hlint/issues/672